### DISCUSSION TITLE
Patreon?

### DISCUSSION THREAD

**Discussion by VengefulAncient** [Rem0o/FanControl.Releases#1412] [Ideas]
Hey, I absolutely love FanControl, it has helped me and my friends **immensely** with our builds, and I'd love to donate to you. However, I really don't want to pay like an 80% transaction fee to PayPal. Ever thought of setting up a Patreon? I'd gladly subscribe to it. Consider it and thank you for your amazing work - this software is nothing short of a game changer and I'll never get tired of promoting it everywhere.

**Comment by Rem0o** [comment]
Github has a built-in platform called "sponsor" that does basically what Patreon does. Look for the sponsor button with a pink heart.
